src/style-property-map.js

```diff
@@ -86,7 +86,23 @@
     if (!propertyDictionary().isSupportedProperty(property)) {
       throw new TypeError('Cannot use has method for ' + property + ' because it is not a supported CSS property');
     }
-    return !!this._styleObject[property]
+    return !!this._styleObject[property];
+  };
+
+  StylePropertyMap.prototype.getProperties = function() {
+    var specifiedPropertyIndex = 0;
+    var isSpecifiedProperty = true;
```

Comment:
This variable doesn't seem to ever change. If this is intentional it is clearer to omit it and write this below:

```javascript
while (true) {
  if (!this._styleObject[specifiedPropertyIndex]) {
```